dateutil/test/test_parser.py

```diff
@@ -11,6 +12,36 @@
 from six.moves import StringIO
 
 
+def test_Ybd_permutations():
```

Comment:
This doesn't actually test anything unless you run it with `pytest` (and even then it's a deprecated way to do this). The function should be moved into the `TestCase` object until we move over to `pytest`, at which point we'll just switch to the parametrized version.

Reply:
Note that by "moved to" I mean that it should not `yield` anything, and the `assert` statement should be switched to a `self.assertEqual()` call.